Restsplain.php

```diff
@@ -38,11 +38,13 @@ function get_default_docs_base() {
  * @return string
  */
 function get_docs_base() {
+	$subdir_install = (string) parse_url( home_url(), PHP_URL_PATH );
+
 	if ( get_query_var( 'restsplain', false ) ) {
-		return get_default_docs_base();
+		return $subdir_install . get_default_docs_base();
```

Comment:
Realised that `get_default_docs_base()` should probably check it starts with a slash 🤔 Don't worry about that for this PR, just a note to myself.